doc_src/index.hdr.in

```diff
@@ -1113,6 +1115,8 @@ function --on-signal WINCH my_signal_handler
 end
 \endfish
 
+Please note that event handlers only become active when a function is loaded which means you might need to manually <a href='commands.html#source'>source</a> a function's file instead of relying on <a href=#syntax-function-autoloading>autoloading</a>.
```

Comment:
I think this could be clearer.

"Functions which are event handlers should be added to the `<a href="index.html#initialization">shell initialization file</a>` to be active with every new shell; as saved functions are `<a href=#syntax-function-autoloading>autoloaded on-demand</a>, the event handlers are not registered in saved functions until the first time they are sourced or executed."

Hmm, I'm not sure that's clearer...